swagger.yaml

```diff
@@ -65,11 +65,11 @@ definitions:
         type: number
         readOnly: true
         description: "The total number of recipes"
-      start:
+      offset:
         type: number
         readOnly: true
-        description: "The start index for the results"
-      items_per_page:
+        description: "The offset index for the results"
+      limit:
         type: number
         readOnly: true
         description: "The number of items per page"
```

Comment:
If you use the following description for `limit` field: `The number of items requested, defaulted to 50 and limited to 1000.`

Reply:
The default value and maximum limit can be implemented as another story/task; again for discussion with Tech leads. I've chosen 50 and 1000 respectively as this matches some of the other API's which implement an items array of objects

Reply:
In the future when the default value and maximum limit are implemented in another story/task, then the description can be amended to accommodate the change